synapse/axon.py

```diff
@@ -1487,6 +1507,47 @@ async def jsonlines(self, sha256, errors='ignore'):
                 raise s_exc.BadJsonText(mesg=f'Bad json line encountered while processing {sha256}, ({e})',
                                         sha256=sha256) from None
 
+    async def unpack(self, sha256, fmt, offs=0):
```

Comment:
Discuss: Do we want to use a default offset or make that a required value by the caller?